packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -679,7 +690,10 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
                 // complete expression inside parens
                 source.slice(dynamicIndex + 1, end),
               )
-              if (!hasViteIgnore) {
+              if (
+                !hasViteIgnore &&
+                !stringIsTemplateLiteralWithNoVars(rawUrl)
```

Comment:
I don't quite understand why a template literal with no variables is triggering a different code path than a template literal with variables. It's true that this the bug is only triggered with no variables today, but it seems a bit strange to have to specifically allow the case with no variables when [the case with variables is also supported](https://vitejs.dev/guide/features.html#dynamic-import) under at least some circumstances. Assuming `stringIsTemplateLiteralWithNoVars` is implemented correctly (which I didn't review), I do think such a check would be sufficient to fix the bug, but am just leaving this comment as a note for someone more familiar with the code to consider.